src/PKSim.Matlab/app.config

```diff
@@ -34,6 +34,22 @@
         <assemblyIdentity name="Microsoft.Extensions.Logging.Abstractions" publicKeyToken="adb9793829ddae60" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-2.1.1.0" newVersion="2.1.1.0" />
```

Comment:
Not sure why we need reference to `Microsoft.Extensions.Logging` in `PKSim.Matlab` if it's not used.